protobuf/model_config.proto

```diff
@@ -1734,6 +1734,30 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+message ListStruct 
```

Comment:
Nest these 3 messages inside of TensorStructure, and we should be consistent and use Structure instead of Struct